<launch>

  <arg name="timeout" default="5" />
  <arg name="left_motor" default="1" />
  <arg name="right_motor" default="2" />
  <arg name="maxspeed" default="0.6" />
  <arg name="minspeed" default="0.1" />
  <arg name="wheelbase" default="0.2" />
  <arg name="turnspeed" default="1.0" />
  <arg name="duty_factor" default="1.0" />
  <arg name="frame_id" default="imu_link" />
  <arg name="linear_acceleration_stddev" default="0.04" />
  <arg name="angular_velocity_stddev" default="0.02" />
  <arg name="orientation_stddev" default="0.0025" />
  <arg name="magnetometer_stddev" default="0.04" />

  <node name="imu_pub_node" pkg="bbblue_drivers" type="imu_pub_node" respawn="true">
    <param name="frame_id" value="$(arg frame_id)" />
    <param name="linear_acceleration_stddev" value="$(arg linear_acceleration_stddev)" />
    <param name="angular_velocity_stddev" value="$(arg angular_velocity_stddev)" />
    <param name="orientation_stddev" value="$(arg orientation_stddev)" />
    <param name="magnetometer_stddev" value="$(arg magnetometer_stddev)" />
  </node>

  <node name="diff_motor_driver" pkg="bbblue_drivers" type="diff_motor_driver" respawn="true">
    <param name="timeout" value="$(arg timeout)" />
    <param name="left_motor" value="$(arg left_motor)" />
    <param name="right_motor" value="$(arg right_motor)" />
    <param name="maxspeed" value="$(arg maxspeed)" />
    <param name="minspeed" value="$(arg minspeed)" />
    <param name="wheelbase" value="$(arg wheelbase)" />
    <param name="turnspeed" value="$(arg turnspeed)" />
    <param name="duty_factor" value="$(arg duty_factor)" />
  </node>

</launch>

